docs/contributing.md

```diff
@@ -121,3 +121,75 @@ Prerequisites: dpkg -i http://archive.ubuntu.com/ubuntu/pool/universe/c/checksec
 Always run make sdl-check command before you submit changes.
 
 Prerequisites: pip3 install bandit
+
+## Logging Policy
+
+### Levels
+
+</br>
+
+#### `ERROR`
+Log at this level errors of every ovms operation that failed 
+</br>
+
+Example:
+```
+    "Failed to create a temporary path PATH"
+```
+
+</br>
+
+#### `WARN`
+Log at this level everything that may lead to an error/need to be fix by administrator.
+</br>
+
+Example:
+```
+    "Expected version directory to be in number format. Got:DIR_NAME"
+```
+
+</br>
+
+#### `INFO` 
+Log at this level all actions that are user-driven, or any others that changes state of application:
+</br>
+
+- processing requests
```

Comment:
Do we want to log **every** request processing at INFO level?
Perhaps it's something that we should do on the DEBUG level?

Reply:
This is what i meant by user-driven :p but youre right - i will move it to the DEBUG level